_source/user-guide/metrics-analytics/configure-grafana-drilldowns.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: article
+title: Configure Grafana drilldown links
+permalink: /user-guide/infrastructure-monitoring/configure-grafana-drilldown-links.html
+flags:
+  logzio-plan: pro
+tags:
+  - metrics
+contributors:
+  - imnotashrimp
+---
+
+One of the most useful features you'll come across
+in Logz.io Infrastructure Monitoring
+is drilldown links.
+With drilldown links,
+if you see something funky,
+you can click a link
+and go straight to the related logs in Kibana.
+
+Drilldown links are preconfigured on any dashboards managed by Logz.io.
+If you're setting up your own dashboard,
+you'll need to configure drilldown links.
+This doc shows you how.
+
+#### To configure Grafana drilldown links
+
+**Before you begin, you'll need**:
+[Metrics]({{site.baseurl}}/shipping/) in your Logz.io metrics account,
+a dashboard that isn't managed by Logz.io
+
+<div class="tasklist">
+
+##### Make a new panel
+
+Open a dashboard that isn't managed by Logz.io (or make a new one).
+
+If you don't see a new panel,
+click the **Add panel** button in the toolbar (in the upper right corner).
+
+![Grafana new panel]({{site.baseurl}}/images/grafana/new-panel.png)
+
+Click **Add Query**.
+
+##### Configure the query and visualization
+
+![Panel configuration, datasource list]({{site.baseurl}}/images/grafana/panel-config--query--datasource-list.png)
+
+In the datasource list, choose the Elasticsearch datasource.
+This is the name of your Infrastructure Monitoring account,
+which you set in the [_Manage accounts_](https://app.logz.io/#/dashboard/settings/manage-accounts) page.
+(By default, it's the name of your main account, followed by "_metrics".)
+
+For help configuring the query,
+see [Query Editor](https://grafana.com/docs/grafana/latest/guides/basic_concepts/#query-editor)
+from Grafana.
+{:.info-box.read}
+
+Click the _Visualization_ icon to the left to see visualization options.
+
+![Panel configuration, visualization  options]({{site.baseurl}}/images/grafana/panel-config--query--visualization.png)
+
+Configure the visualization as you normally would.
+
+##### Configure the drilldown
+
```

Comment:
Sorry I don't see this row here either (I guess I'm not familiar enough with github) but this row is inaccurate:

![image](https://user-images.githubusercontent.com/55798673/70718903-c96cb300-1cf9-11ea-9689-3c30bb784ab2.png)
It's very confusing, but the query in the panel is not related to the query in the drilldownlink. You have to write your query again in the Url params box as" query=....", and the two queries might differ from each other.
So you have one query for the panel and then you make another one for the drilldownlink.

Reply:
Thanks for the clarification! Fixed in the next push